third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -134,28 +134,29 @@ func TestAccContainerCluster_withAddons(t *testing.T) {
 	t.Parallel()
 
 	clusterName := fmt.Sprintf("tf-test-cluster-%s", randString(t, 10))
+	pid := getTestProjectFromEnv()
```

Comment:
I'd use "project", unless this is in use elsewhere in the file.

Reply:
pid was used in other tests, just used the same to be consistent